src/Symfony/Component/Messenger/Bridge/AmazonSqs/CHANGELOG.md

```diff
@@ -6,3 +6,8 @@ CHANGELOG
 
  * Introduced the Amazon SQS bridge.
  * Added FIFO support to the SQS transport
+
+5.1.3
```

Comment:
5.2.0 :) This should be moved to the first position also, at the beginning of the file

Reply:
thanks!